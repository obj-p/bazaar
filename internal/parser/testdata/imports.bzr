package imports

import foo.bar.baz
import foo.bar.baz.Qux
// TODO: support alias
// import bar.baz.Qux as Foo
