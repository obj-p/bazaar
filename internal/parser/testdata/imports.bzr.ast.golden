&parser.Bazaar{
  Package: &parser.PackageDecl{
    Domain: []string{
      "imports",
    },
  },
  Imports: []*parser.ImportDecl{
    {
      Domain: []string{
        "foo",
        "bar",
        "baz",
      },
    },
    {
      Domain: []string{
        "foo",
        "bar",
        "baz",
        "Qux",
      },
    },
  },
}
